From dad4a67eb3622bda71c5ea3f184fe02722bc67c1 Mon Sep 17 00:00:00 2001
From: Sam Thursfield <sam.thursfield@codethink.co.uk>
Date: Fri, 5 Jun 2026 11:43:17 +0100
Subject: [PATCH 02/19] FDSDK 26.08: Update systemd-base.bst for removed
 components/pyelfutils.bst element

This element was duplicated in FDSDK under two names, in commit 1d302725239ee0b414f669e1f
upstream removed the one we were using.
---
 elements/core-deps/systemd-base.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/core-deps/systemd-base.bst b/elements/core-deps/systemd-base.bst
index 03571348c..c39967379 100644
--- a/elements/core-deps/systemd-base.bst
+++ b/elements/core-deps/systemd-base.bst
@@ -39,7 +39,7 @@ build-depends:
 - freedesktop-sdk.bst:components/libarchive.bst
 - freedesktop-sdk.bst:components/linux-pam.bst
 - freedesktop-sdk.bst:components/kmod.bst
-- freedesktop-sdk.bst:components/pyelftools.bst
+- freedesktop-sdk.bst:components/python3-pyelftools.bst
 - freedesktop-sdk.bst:components/libxslt.bst
 - freedesktop-sdk.bst:components/docbook-xsl.bst
 - freedesktop-sdk.bst:components/cryptsetup-lvm2-stage1.bst
-- 
2.54.0

